### DISCUSSION TITLE
It keep promiting Username and Password

### DISCUSSION THREAD

**Discussion by HassamChaudhary1** [community/community#57594] [New to GitHub]
I'm facing an issue while downloading the spec repo from GitHub. I don't understand why it's keep asking it even the doc link is not working. I searched and try a lot of different way to prevent this credential but it keeps asking.

**Link :** remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.

<img width="998" alt="Screenshot 2023-06-09 at 11 49 29 AM" src="https://github.com/community/community/assets/79073135/8099fffd-0ab4-4343-be4e-0c8b81aa4b24">

**Comment by TithiB-del** [comment]
I think you have not set up git in your device correctly.
you need to set your username , email & password to be able to do things smoothly.
https://docs.github.com/en/get-started/quickstart/set-up-git
to be able to clone a repo, ur account needs to be set up correctly, & the remote file would be saved in your local device storage.

**Comment by airtower-luna** [comment]
You cannot use your password for `git fetch` or other Git operations on GitHub (but you still need it to log in). You need to [create a personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) and enter that instead when Git asks for your password. When creating the token make sure to select the **repo** scope (for repository access, like pushing) and if you use Actions also **workflow** (required to modify workflow files).
